base/sort.jl

```diff
@@ -478,7 +478,7 @@ for (sym, exp, type) in [
     @eval function $usym(v, o, kw)
         # using missing instead of nothing because scratch could === nothing.
         res = get(kw, $(Expr(:quote, sym)), missing)
-        res !== missing && return kw, res::$type
+        !ismissing(res) && return kw, res::$type
```

Comment:
This check should remain egality. We only want to compute the default value for `sym` if it is not already in `kw`, in which case res will be literally egal to `missing`.